### ISSUE TITLE
Matching glyphs sets

### ISSUE THREAD

**Issue by vv-monsalve** [vv-monsalve/space-grotesk#4]
Please, ensure the set of glyphs on Space Grotesk also includes the ones already present on Space Mono. Test performed through Wakami and Diffenator reports a difference in ~200 characters. 

Counted by glyphs the difference could be bigger due to alternate sets, but at least it should include the same characters, e.g.,  acircumflextilde or 'onethird' fraction.

**Comparative before&after gif between fonts.** 
The image works only as a reference since it also displays glyphs formed by combining marks. E.g., Hcircumflex, which doesn't exist as a pre-built compound glyph in Space Grotesk, and therefore, it also shows a miss position of the diacritic.

![glyphs_missing](https://user-images.githubusercontent.com/48698976/81028437-02dbc180-8e47-11ea-9444-5498b9eb4426.gif)

**Comment by mightybart** [comment]
I'll start with Vietnamese (Ohorn, ohorn, Uhorn, uhorn, hookabove, accents positioning).

**Comment by mightybart** [comment]
Current sate:

Space Mono
<img width="1153" alt="Screenshot 2020-07-02 23 18 17" src="https://user-images.githubusercontent.com/9280418/86410185-9d774600-bcba-11ea-808a-79580e8c005b.png">

Space Grotesk
<img width="1154" alt="Screenshot 2020-07-02 23 18 58" src="https://user-images.githubusercontent.com/9280418/86410202-a5cf8100-bcba-11ea-9518-bf5819b6d547.png">

Missing characters (will fix asap): Ccircumflex, ccircumflex, Ebreve, ebreve, Gcircumflex, gcircumflex, Gcaron, gcaron, Hcircumflex, hcircumflex, Ibreve, ibreve, Jcircumflex, jcircumflex, kgreenlandinc, napostrophe, Obreve, obreve, Oogonek, oogonek, Scircumflex, scircumflex, Schwa, schwa

**Comment by mightybart** [comment]
Space Grotesk after update:

<img width="1152" alt="Screenshot 2020-07-03 00 14 46" src="https://user-images.githubusercontent.com/9280418/86413785-74f34a00-bcc2-11ea-8e7f-d6428a4b6bff.png">

**Comment by vv-monsalve** [comment]
After reviewing the files and running Diffenator, there are still some missing glyphs.
The ligatures were missing the Unicode, those were already fixed. Please check the others cases.

![glyphs_missing](https://user-images.githubusercontent.com/48698976/87578862-aee92680-c69a-11ea-865d-62b112408e5a.gif)

**Comment by mightybart** [comment]
Fixed in my [v2 branch](https://github.com/floriankarsten/space-grotesk/tree/v2). Latest QA report below: these are just some stylistic alternates/capital versions. Dutch ÍJ is also allright, there is 'locl' feature for that. I decided to skip the upside down/flipped characters completely, as there is not even full set included in Space Mono.

![glyphs_missing](https://user-images.githubusercontent.com/9280418/88352164-254fed80-cd59-11ea-8463-4ef15d5bbd11.gif)
